SRC/NBASIC/nbas111e.txt

```diff
@@ -827,11 +822,11 @@ ensemble use of NestorBASIC and InterNestor Lite.
 9.5. ERRORS
 
 Function 86 (InterNestor Lite routine execution)returns error -1 if 
-InterNestor Lite is not installed.
+InterNestor Lite is not resident-loaded.
 
 Function 85 (InterNestor Suite routine execution) returns error -1 if 
-InterNestor Suite is not installed, or if an invalid module number is 
-specified in P(0) (valid segment numbers are 1 to 4).
+InterNestor Suite is not loaded, or if an invalid module number is 
+specified in P(0) (valid module numbers are 1 to 4).

```

Comment:
I think "module" in the previous line was a mistake, it should be "segment" in both.

Reply:
Nope, it's "module".
![Internestor_Suite_module_number ](https://user-images.githubusercontent.com/30573655/143271241-0cf7ae23-8f19-473e-8e2b-626b1c15ec8b.png)

Reply:
True. That was like 20 years ago and I had forgotten. 😄